src/js/play.js

```diff
@@ -268,7 +269,7 @@ var playState = {
         //Stopping eggs from falling by pausing all the time events loop to begin explosion animation
         game.time.gamePaused();
         gameController.player.animations.play('explodeBomb');
-        gameController.player.inputEnabled = false;
```

Comment:
Without the above line you can move the basket while the explosion animation is playing. 
Not sure if this is a bad thing to have or low-key comical.